lib/CSPUpdater.js

```diff
@@ -0,0 +1,50 @@
+
+var jsdom = require('jsdom');
+var fs = require('fs');
+var Q = require('q');
+var path = require('path');
+
+// This snippet grants all permissions necessary for the app to access the BrowserSync Server and thus enables us to do LiveReload
+var CSP_SNIPPET = "\n\n\n<!-- BEGIN -- Added by cordova-plugin-livereload--> \n<meta http-equiv=\"Content-Security-Policy\" content=\"default-src *; style-src * 'unsafe-inline' 'unsafe-eval'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *;\"> \n <!-- END -- Added by cordova-plugin-livereload -->\n\n\n";
+
+module.exports = function(indexHTMLPage) {
+    
+    this.indexHTMLPage = indexHTMLPage;
+    this.indexHTMLContent = fs.readFileSync(indexHTMLPage, "utf-8");
+    
+    var self = this;
+    this.Update = function() {
+
+	var deferred = Q.defer();
+	var jquery = fs.readFileSync(path.join(__dirname, "..", "jquery", "./jquery-1.11.1.js"), "utf-8");
+
+	jsdom.env(
+    	    {
+		html: self.indexHTMLContent,
+		src: [jquery],
+		done: function (err, window) {
+		    var $ = window.$;
+		    
+		    // Delete the CSP tag included by the current app
+		    $("meta[http-equiv=Content-Security-Policy]").remove();
```

Comment:
Looks like we are replacing the CSP policy - should we instead edit it and add the attributes that we care about? Alternatively, if we go down this route - one option is to just remove the CSP policy instead of editing it.